unity-renderer/Assets/UIComponents/Scripts/Components/DynamicIconList/DynamicListComponentView.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DynamicListComponentView : BaseComponentView, IDynamicListComponentView
+{
+    [SerializeField] private Image iconReference;
+
+    internal List<Image> instantiatedIcons = new ();
+    public override void RefreshControl()
+    {
+    }
+
+    public void AddIcon(Sprite sprite)
+    {
+        Image newImage = Instantiate(iconReference, transform);
+        newImage.sprite = sprite;
+        instantiatedIcons.Add(newImage);
+    }
+
+    public void AddIcons(List<Sprite> spriteList)
+    {
+        RemoveIcons();
+
+        foreach (var sprite in spriteList)
+        {
+            Image newImage = Instantiate(iconReference, transform);
+            newImage.sprite = sprite;
+            instantiatedIcons.Add(newImage);
```

Comment:
We can reuse the `AddIcon` method here, right?
```suggestion
            AddIcon(sprite);
```